docs/spectator/lang/java/ext/jvm-classloading.md

```diff
@@ -0,0 +1,39 @@
+# ClassLoading
+
+Uses the [ClassLoadingMXBean](https://docs.oracle.com/javase/7/docs/api/java/lang/management/ClassLoadingMXBean.html)
+provided by the JDK to monitor the number of classes loaded and unloaded.
+
+## Getting Started
+
+To get information about classloading in Spectator, just setup registration of standard MXBeans.
+Note, if you are building an app at Netflix, then this should happen automatically via the normal
+platform initialization.
+
+```java
+import com.netflix.spectator.api.Spectator;
+import com.netflix.spectator.jvm.Jmx;
+
+Jmx.registerStandardMXBeans(Spectator.registry());
```

Comment:
[Spectator.registry()](https://www.javadoc.io/static/com.netflix.spectator/spectator-api/1.3.0/com/netflix/spectator/api/Spectator.html#registry--) is deprecated. You could use `Spectator.globalRegistry()`, but I think if they are getting here they should know the registry instance and it could just be:

```java
import com.netflix.spectator.jvm.Jmx;

Jmx.registerStandardMXBeans(registry);
```

Reply:
Fixed for all instances.